src/components/dashboard/roaster/RoasterAccount.js

```diff
@@ -33,7 +38,11 @@ class RoasterAccount extends Component {
             addressCountry: this.country.value
         };
 
-        dispatch(updateRoaster(data, this.props.roaster.id));
+        dispatch(updateRoaster(data, this.props.roaster.id)).then(() => {
```

Comment:
see above comment about `.then`. I haven't seen any docs on it, so I'm not positive.